PU: Visualizar perfil de usuario-#11

javsolgar opened the pull request:
Se ha implementado la consulta de un perfil por parte de un usuario y se han generado pruebas para comprobar su correcto funcionamiento Closes #11 .

Por otra parte, se ha realizado una pequeña modificación en el test de publicación para solucionar un problema de consulta a la base de datos #9 .

rafsalcas1 committed:
Adicción de perfil usuario-#11

javsolgar committed:
Finalización Perfil-#11

Se han realizado unas pequeñas modificaciones en el view.py, se han generado las pruebas de Perfil y se ha realizado una modificación para el correcto funcionamiento de las pruebas de Publicación #9

fragonlug reviewed (approved):
Se ha observado que se lista las publicación correctamente asociadas al usuario, estando pendientes su correcta visualización, teniendo en cuenta que deberían aparecer primero las más reciente, pero se comprende que forma parte del equipo de frontend.

mrf1989 commented on youroom/templates/perfil/perfil.html:
```diff
@@ -8,11 +8,11 @@
                 <section class="row col-12">
                     <div>
-                        <p>Diseñador/a de interiores <br> Creating a happy home <br> Contacto: diseño@diseño.com <br> Mi tienda: miTienda.com</p>
+                        <p>{{usuario.descripcion}} <br> Contacto: diseño@diseño.com <br> Mi tienda: miTienda.com</p>
```
No entiendo estos cambios ni tampoco estos estáticos como "contacto" o "mi tienda".

Además, insisto, aquí se le está metiendo mano a parte de frontend en una tarea de backend. Cuidado porque al desplegar la app y navegar a /perfil las publicaciones se rompen y las etiquetas ya no aparecen correctamente.

javsolgar replied:
Según he hablado con mis compañeros, lo de la tienda es un residuo que ya estaban en la template y que no quisieron cambiar para no molestar a los de frontend. Ellos solo quisieron probar si funcionaba el código que hicieron en backend. Igualmente si lo ves adecuado, podemos dejar simplemente el contenido que hay en develop.

mrf1989 replied:
@javsolgar Entiendo que, a veces, programar el backend es complicado porque se necesita tener una confirmación visual de que la implementación es correcta y hace lo que se espera. También para ello están los tests. Pero si de todos modos los tests fueran insuficientes y no se tiene la seguridad de que lo que está hecho está bien sin verlo, ok, se puede tocar el frontend, pero no commitear esa parte.

Finalmente esos archivos no deberían formar parte del commit, ya que podría trastocar trabajo hecho y consumir un tiempo de trabajo que debería emplearse solo a integrar o resolver la integración.

javsolgar replied:
De acuerdo, tomamos nota y realizamos un nuevo commit a la rama cumpliendo con los cambios que nos has solicitado. ¡Muchas gracias!

mrf1989 reviewed (changes requested):
La parte frontend no se debería tocar ni verse afectada por los progresos. Lo que se avance de un lado no puede ir en detrimento de otro.

Metedle una revisada, por favor.

javsolgar committed:
Se ha restaurado perfil.html-#11

Se ha copiado el contenido original de perfil.html en la rama develop, para que así este se enc
uentre en sus estado original.

javsolgar commented:
Se ha dejado el contenido de perfil.html exactamente igual que en develop.
